patches/api/0352-Added-EntityClimbEvent.patch

```diff
@@ -0,0 +1,48 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Jan Trummer <jan.trummer@hotmail.com>
+Date: Mon, 3 Jan 2022 18:06:18 +0100
+Subject: [PATCH] Added EntityClimbEvent
+
+
+diff --git a/src/main/java/io/papermc/paper/event/entity/EntityClimbEvent.java b/src/main/java/io/papermc/paper/event/entity/EntityClimbEvent.java
+new file mode 100644
+index 0000000000000000000000000000000000000000..7aa1ee22171d6eb049e80e41cc88d36caa36bd4f
+--- /dev/null
++++ b/src/main/java/io/papermc/paper/event/entity/EntityClimbEvent.java
+@@ -0,0 +1,36 @@
++package io.papermc.paper.event.entity;
++
++import org.bukkit.block.Block;
++import org.bukkit.entity.LivingEntity;
++import org.bukkit.event.Cancellable;
++import org.bukkit.event.HandlerList;
++import org.bukkit.event.entity.EntityEvent;
++import org.jetbrains.annotations.NotNull;
++
++public class EntityClimbEvent extends EntityEvent{
++
++    private static final HandlerList HANDLER_LIST = new HandlerList();
++    private Block climbableBlock;
++
++    public EntityClimbEvent(@NotNull LivingEntity entity, Block climbableBlock) {
++        super(entity);
++        this.climbableBlock = climbableBlock;
++    }
++
++    @NotNull
++    @Override
++    public HandlerList getHandlers() {
++        return HANDLER_LIST;
++    }
++
++    @NotNull
++    public static HandlerList getHandlerList() {
++        return HANDLER_LIST;
++    }
++
++    @NotNull
++    public Block getClimbableBlock()
++    {
++        return climbableBlock;
++    }
```

Comment:
Please stick to the same style, the opening bracket shouldn't be on a separate line.